samples/v1.3/ElementsPreview/Input.Text.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.3",
+	"body": [
```

Comment:
These are meant to replace/override the files currently in the v1.0/elements folder, right? I noticed that some of these new files tend to be more validation focused, where the older files showed a wider variety of the properties available on each type. For example, the v1.0\Elements\Input.Text.json file demonstrates the `maxLength`, `isMultiline`, and `value` properties. Was that change intentional? This card in particular looks like a good sample for the isRequired property, but is that the main/only thing we want to highlight in our primary Input.Text sample card on the website?

Reply:
agree, I think this may seem like a duplicate of https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.3/Elements/Input.Text.Label.json

Reply:
Now reflects the existing samples.